src/main/java/net/openhft/hashing/CharSequenceAccess.java

```diff
@@ -114,12 +143,17 @@ private BigEndianCharSequenceAccess() {}
 
         @Override
         public long getLong(CharSequence input, long offset) {
-            return getLong(input, offset, 3, 2, 1, 0);
+            return getLong(input, offset, 3, 2, 1, 0, 0, 1);
```

Comment:
Looks strange that this is not as symmetric as little-endian, logically there should be -1 after 0. Did you test this?

Reply:
the two new arguments are only used when offset is misaligned:
- the `Off` of the highest char when offset is unalign, i.e. `char4Off` for `getLong()`
- the delta for other chars

For example, in little-endian `getLong()`

- char0: align=0 unalign=0 + **0**
- char1: align=1 unalign=1 + **0**
- char2: align=2 unalign=2 + **0**
- char3: align=3 unalign=3 + **0**
- char4:                unalign=**4**

in big-endian `getLong()`

- char0: align=3 unalign=3 + **1**
- char1: align=2 unalign=2 + **1**
- char2: align=1 unalign=1 + **1**
- char3: align=0 unalign=0 + **1**
- char4:                unalign=**4**


All `Access` public methods have passed the ut in `CharSequenceAccessTest.java` for both little- and big- endians.

Reply:
Ok